UI/MVC.js

```diff
@@ -0,0 +1,327 @@
+const users = [{ login: 'janny_9991', password: '123456' }, { login: 'katya', password: 'katya1' }, { login: '2', password: '2' }];
+
+function randomInteger(min, max) {
+  let rand = min - 0.5 + Math.random() * (max - min + 1);
+  rand = Math.round(rand);
+  return rand;
+}
+
+class MVC {
+  constructor(model, view) {
+    this.model = model;
+    this.view = view;
+  }
+
+  showMorePosts() {
+    const button = document.querySelector('.show-more-posts');
+    let count = 0;
+    button.addEventListener('click', () => {
```

Comment:
addEL should be in a bind function that you call on init, but the handler itself (callback) should be named and separated